wallet-new/src/Cardano/Wallet/API/V1/Migration/Types.hs

```diff
@@ -129,6 +130,10 @@ instance Migrate (V1.WalletId, V1.AccountId) V0.AccountId where
     eitherMigrate (walId, accId) =
         V0.AccountId <$> eitherMigrate walId <*> pure accId
 
+instance Migrate (V1.WalletId, V1.AccountId) V0.CAccountId where
+    eitherMigrate (walId, accId) =
+        V0.encodeCType <$> eitherMigrate (walId, accId)
```

Comment:
If you write the complementary `Migrate` instances (i.e from `V0.CAccountId` to `(V1.WalletId, V1.AccountId)`) you can then add a roundtrip test.

Reply:
If I remember you correctly we don't want to a `Migrate` counter part for testing only. Because currently there is no need to have `Migrate V0.V0.CAccountId ((V1.WalletId, V1.AccountId)`

Reply:
No strong feelings here, i'm mentioning this as @akegalj was very keen on having counterparts written for the sake of testing, but I'm fine with sticking with the status quo 😉